content/unsupported-browser.html

```diff
@@ -4,15 +4,28 @@
 robots_tag: noindex
 ---
 
+<script defer src="/assets/js/unsupported-browser.js"></script>
+
 <section id="intro" class="intro-heading">
   <h1 class="mb-3">Unsupported Browser</h1>
   <p class="mb-1">
     Scratch Addons doesn't support the browser you're using.
   </p>
 </section>
 
-<section class="container">
-  <p>Use updated versions of Chrome, Firefox, or other browsers based on them to get started:</p>
+<div class="container">
+
+<section class="mb-5">
+  <p>Your browser is not supported by Scratch Addons.</p>
+  <div id="export-settings" style="display: none">
+    <button id="export-btn" class="btn btn-dark dark-btn-white">Export my extension settings</button>
+    <button id="export-view-btn"class="btn btn-dark dark-btn-white">View my extension settings</button>

```

Comment:
Additional context: these two buttons are supposed to work just like the ones in the extension settings page.
![image](https://github.com/ScratchAddons/website-v2/assets/17484114/ee81bacb-6a63-4a97-b9a2-94494420b236)